release/community-stable/oraclelinux/docker/Dockerfile

```diff
@@ -5,7 +5,7 @@
 # installed from RHEL7 PowerShell package
 
 # Define arg(s) needed for the From statement
-FROM oraclelinux:latest AS installer-env
```

Comment:
Add the file to the depandabot config

Reply:
hmmm.... I can add a dockerfile in a dependabot folder but how do you add the config? Where is this `/.dependabot/config.yml` file?